cli/ops/os.rs

```diff
@@ -176,6 +178,16 @@ fn op_hostname(
   Ok(JsonOp::Sync(json!(hostname)))
 }
 
+fn op_type(
+  state: &State,
+  _args: Value,
+  _zero_copy: Option<ZeroCopyBuf>,
+) ->  Result<JsonOp, OpError> {
+  state.check_env()?;
+  let info = uname().unwrap().sysname;
+  Ok(JsonOp::Sync(json!(info)))
```

Comment:
Can you use libc::uname directly, rather than adding this wrapper crate?

Reply:
> Can you use libc::uname directly, rather than adding this wrapper crate?

I can try.